include/zephyr/pm/policy.h

```diff
@@ -58,6 +58,23 @@ struct pm_policy_latency_request {
 	/** @endcond */
 };
 
+/**
+ * @brief Callback to notify subscribers of the next event.
+ *
+ * @param next_event Next event in cycles.
+ */
+typedef void (*pm_policy_next_event_cb_t)(int64_t next_event);
+
+/**
+ * @brief Next event subscription.
+ *
+ * @note All fields in this structure are meant for private usage.
+ */
+struct pm_policy_event_subscription {
+	sys_snode_t node;
+	pm_policy_next_event_cb_t cb;
+};
```

Comment:
```suggestion
/**
 * @brief Next event subscription.
 */
struct pm_policy_event_subscription {
	/**
	 * @cond INTERNAL_HIDDEN
	 */
	sys_snode_t node;
	pm_policy_next_event_cb_t cb;
	/**
	 * @endcond
	 */
};
```

Reply:
@siddharthgpta not sure if this is an oversight, but fyi your most recent push doesn't seem to have addressed this (in fact, it's not clear if it contains any change at all :) )

Reply:
Yes looks like I messed up the branch I pushed to. Thanks for flagging it again, I'll fix it!